lib/import/admin.js

```diff
@@ -53,6 +55,9 @@ function enrich_features(country_code, feature_collection, verbose) {
       feat.properties.admin_code = country_code + '-' +
         (feat.properties.CHIE_CODE || feat.properties.ID_2);
 
+      // Assign population to admin
+      feat.properties.population = admin_pop[feat.properties.admin_code];
```

Comment:
Could you check that this value is actually defined before setting it? No point in storing `undefined` values in Mongo and our TopoJSON.